tools/etcd-tls-setup/README.md

```diff
@@ -0,0 +1,22 @@
+# etcd-tls
+
+## Overview
+
+This image provide tools and scripts to generate TLS certificates for communication between service catalog and etcd.
```

Comment:
`This image provides tools and scripts to generate TLS certificates for communication between the Service Catalog and etcd.`